src/LegendRenderer/SvgOutput.ts

```diff
@@ -49,22 +62,23 @@ export default class SvgOutput extends AbstractOutput {
       .attr('text-anchor', 'start')
       .attr('dx', x)
       .attr('dy', y);
-  };
+  }
 
-  addLabel(text: string, x: number | string, y: number | string) {
+  addLabel(text: string, x: number | string, y: number | string, legendItemTextSize: number | undefined) {
     this.currentContainer?.append('text')
       .text(text)
       .attr('x', x)
-      .attr('y', y);
+      .attr('y', y)
+      .style('font-size', legendItemTextSize + 'px');
```

Comment:
Should we check for undefined of legendItemTextSize here? Otherwise it might set font-size to `undefinedpx` right?